example/testapp/models.py

```diff
@@ -9,10 +9,13 @@ class Car(models.Model):
     def __unicode__(self):
         return self.plate
 
+class m2mprint:
+    def __unicode__(self):
```

Comment:
I suggest you to move this new model to:
https://github.com/gregmuellegger/django-sortedm2m/blob/master/sortedm2m_tests/models.py

It will be easier for you to import that model in the test suite.

In the test suite you will have to ensure the representation of the model is what you expect and I'm pretty confident declaring __unicode__ only would not work on python3.
I leave the solution up to you, as long as the test works, I'm fine with it.

I pointed out that solution because in the OpenWISP modules I try to use `python_2_unicode_compatible` as much as possible but this package was probably developed before this utility was added.